airflow/models/connection.py

```diff
@@ -187,10 +187,22 @@ def _normalize_conn_type(conn_type):
         return conn_type
 
     def _parse_from_uri(self, uri: str):
+        schemes_count_in_uri = uri.count("://")
+        if schemes_count_in_uri > 2:
+            raise AirflowException(f"Invalid connection string: {uri}.")
+        scheme_in_uri = schemes_count_in_uri == 2
```

Comment:
This variable name is rather confusing to me, can we have something else here?

Reply:
can you check it now?